src/components/app/pageEvents/components/stateEventsDialogContent.tsx

```diff
@@ -39,30 +43,33 @@ export function StateEventsDialogContent({ state, events }: StateEventsDialogPro
         {US_MAIN_STATE_CODE_TO_DISPLAY_NAME_MAP[parsedState]}.
       </p>
 
-      {stateEvents && stateEvents?.length > 0 && (
-        <div className="mt-6 flex w-full flex-col gap-4 overflow-y-auto px-8 lg:max-h-96">
-          {stateEvents.map(event => (
-            <EventDialog
-              event={event.data}
-              key={event.data.slug}
-              trigger={<StateDialogEventCard event={event.data} />}
-            />
-          ))}
-        </div>
-      )}
+      <ScrollArea>
+        {stateEvents && stateEvents?.length > 0 && (
```

Comment:
This should be enough:

```suggestion
        {stateEvents?.length && (
```

Reply:
![image](https://github.com/user-attachments/assets/20c6066e-0585-4151-b373-e36f8fd44d58)
Don't do that, `stateEvents?.length > 0` is the most compact and safe way to do it

Reply:
You are right, did not see we were using logical values for rendering. This is a big issue with using logical operators for rendering stuff, way too common of a mistake when using non boolean values. I would rather always using `condigiton ? component : null` when conditionally rendering components because you avoid falling on this trap.

Reply:
Ok!